aspnetcore/client-side/import-export-interop.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Run .NET from JavaScript using `[JSImport]`/`[JSExport]` interop
+author: pavelsavara
+description: Learn how to run .NET from JavaScript using `[JSImport]`/`[JSExport]` interop.
+monikerRange: '>= aspnetcore-7.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 10/18/2022
+uid: client-side/import-export-interop
+---
+# Run .NET from JavaScript using `[JSImport]`/`[JSExport]` interop
+
+This article explains how to run .NET from JavaScript (JS) using `[JSImport]`/`[JSExport]` interop.
```

Comment:
It might be good to clarify that "import" and "export" here are defined from the perspective of .NET. You import JS methods so they can be called from .NET and you export .NET methods so they can be called from JS.